Makefile

```diff
@@ -11,6 +11,9 @@ help:
 build: dep ## Build for development only
 	 go build -o ./${OUTPUT}/${EXECUTABLE} ./cmd/${EXECUTABLE}
 
+build-linux: dep ## Build binary for linux
+	GOOS=linux GOARCH=amd64 go build ${LDFLAGS} -o ${OUTPUT}/${EXECUTABLE}-linux-amd64 ./cmd/${EXECUTABLE}
+
 build-docker: ## Builds Docker with linux lenses-cli
 	docker build -t landoop/lenses-cli:${VERSION} .
```

Comment:
We should start adopting the `lensesio` namespace instead of `landoop`

Reply:
I will add it to the scripts